components/lsp/MintForm.tsx

```diff
@@ -53,11 +58,21 @@ const MintForm: FC<Props> = ({
   shortTokenBalance,
   refetchLongTokenBalance,
   refetchShortTokenBalance,
+  collateralSymbol,
+  showWallet,
+  setShowWallet,
 }) => {
-  const [collateral, setCollateral] = useState("uma");
+  const [collateral, setCollateral] = useState(collateralSymbol);
   const [amount, setAmount] = useState("");
   const [longTokenAmount, setLongTokenAmount] = useState("");
   const [shortTokenAmount, setShortTokenAmount] = useState("");
+  const { signer } = useConnection();
+
+  useEffect(() => {
+    if (signer) {
+      setShowWallet(false);
+    }
+  }, [signer]);
```

Comment:
`setShowWallet` is a missing dep, it missing is not an issue now as it is just a `setState`, but if we change the impl then it might be an issue